main.go

```diff
@@ -34,9 +33,6 @@ func main() {
 	if v := os.Getenv("BABBAGE_URL"); len(v) > 0 {
 		config.BabbageURL = v
 	}
-	if v := os.Getenv("RESOLVER_URL"); len(v) > 0 {
```

Comment:
`RESOLVER_URL` should also be removed from the README.

Reply:
done